### DISCUSSION TITLE
@click in ajax returned html not working

### DISCUSSION THREAD

**Discussion by TerrePorter** [alpinejs/alpine#2938] [1. Help] [answered]
Hi all,

I am trying to use a @click on a li list of results loaded from livewire that opens a modal.

Since the HTML is loaded dynamically the alpine doesn't process the new HTML code. So my problem is that the @click doesn't work.

Is there a way to fix this issue?

**Comment by SimoTod** [comment]
Can you post an example? 

P. S. `@click` is a user if you don't put it in backticks, he gets notifications. 😂

**Reply by TerrePorter** [reply]
@SimoTod 
There is not much code to share. It is just a typical search box that sends a query to the server using livewire, that returns this HTML, in that HTML there is a list of businesses.

```
// return HTML list of businesses
<div class="w-full bg-white rounded-lg shadow-lg mb-5 overflow-auto h-32">
    <ul class="divide-y-2 divide-gray-400">
        ...
        <li class="flex justify-between p-3 hover:bg-blue-600 hover:text-blue-200" @click="selectBusiness(8424, 'The Pit Stop', 'Dunedin, fl')">
            <div>
                The Pit Stop
            </div>
            <div>
                Dunedin, fl
            </div>
        </li>
        ...
    </ul>
</div>
```

That returned HTML contains the @click on each li. That in a nutshell is the flow.

**Reply by SimoTod** [reply]
I asked because it seems to work for me. I assume you don't have errors in console. Are you using the latest version? If you are using x-html, I believe there was a bug fixed a few releases ago affecting it.

**Comment by TerrePorter** [comment, marked as answer]
Nevermind it was a type in one of my files.
